src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -952,14 +953,24 @@ private StringBuffer preMigrationChecks(StringBuffer itemStatus,
 		 */
 		@Async
 		protected Future<String> uploadBoxFile(MigrationBoxFile bFile, HttpContext httpContext, String sessionId) {
+			// status string
+			StringBuffer status = new StringBuffer();
 			
 			// get all bFile params
 			String id = bFile.getId();
 			String userId = bFile.getUser_id();
 			String type = bFile.getType();
 			String boxFolderId = bFile.getBox_folder_id();
 			String fileName = bFile.getTitle();
-
+			
+			if (fileName == null || fileName.isEmpty())
+			{
+				status.append(" uploadFile: file name is null or empty. ");
+				// update job end time and status
```

Comment:
i think this case can be logged here, not sure `status` is getting logged to server logs some where else in the code